pagopa/io-app
# chore: [IOBP-1060] Add payment authorization generic error screen

## Short description
This PR handles the generic error when performing the authorization request before showing the in-app browser with a specific error screen.

## List of changes proposed in this pull request
- Added the `AUTH_REQUEST_ERROR` custom outcome to handle any request error during the payment authorization request;

## How to test
- Run the dev-server commenting the entire implementation of the API `/transactions/:transactionId/auth-requests`
- Start the payment flow until you're in the confirmation screen that appears after you choose a payment method and the PSP;
- Check that the error screen is the same as [the designed one](https://www.figma.com/design/KY9BpP4LaSSd18fuYMwxYK/Paga-un-avviso-pagoPA?node-id=9631-7305&t=M7R1fYXEWqwxpWpe-4);

## Preview

https://github.com/user-attachments/assets/f5c42050-a8fc-4db7-8c20-43aa7e4202d0
